modules/google-pubsub/src/parse-pubsub-message.pipe.test.ts

```diff
@@ -0,0 +1,60 @@
+import {PubsubService} from './pubsub.service';
+import {PushMessage} from './domain';
+import {ParsePubsubMessagePipe} from './parse-pubsub-message.pipe';
+import {BadRequestException, HttpException} from '@nestjs/common';
+
+describe('ParsePubsubMessagePipe', () => {
+    let mockPubsubService: PubsubService;
+    let parsePubsubMessagePipe: ParsePubsubMessagePipe<string>;
+
+    beforeEach(() => {
+        mockPubsubService = <any> {
+            decryptMessage: jest.fn().mockReturnValue('{}'),
+            verifyMessage: jest.fn().mockReturnValue(true),
+        };
+        parsePubsubMessagePipe = new ParsePubsubMessagePipe(mockPubsubService);
+    });
+
+    test('transform should verify the signature', async () => {
```

Comment:
there is no reason to put method name to the test case name, pls remove `transform ` from all names

also reading test case after is simpler: _test should verify ..._